docs/components/motor/tmc5072.md

```diff
@@ -14,9 +14,6 @@ This model supports stepper motors controlled by the [TMC5072 chip](https://www.
 
 Whereas a basic low-level stepper driver supported by the [`gpiostepper` model](/components/motor/gpiostepper/) sends power to a stepper motor based on PWM signals from GPIO pins, the TMC5072 chip uses SPI bus to communicate with the board, does some processing on the chip itself, and provides convenient features including StallGuard2<sup>TM</sup>.
 
-To configure a `TMC5072` motor as a component of your robot, first configure the [board](/components/board/) to which the motor driver is wired.
```

Comment:
Is that no longer a requirement?

Reply:
Right. The TMC motor component now talks directly to the SPI bus without needing to go through the board component.